registration/api/v2/serializers.py

```diff
@@ -288,6 +306,17 @@ class OpenEdXInstanceConfigSerializer(serializers.ModelSerializer):
     public_contact_email = serializers.EmailField(required=False)
     is_email_verified = serializers.BooleanField(source='email_addresses_verified', read_only=True)
 
+    def _validate_subdomain_not_contains_reserved_word(self, subdomain):
```

Comment:
I'll adjust it soon.

Reply:
I decided to use staticmethod decorator here, since we have no other places where we would like to validate subdomains. (except if we decide to use that on model validator level as well)